content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -81,6 +81,7 @@ different Kubernetes components.
 | `CSIMigrationGCEComplete` | `false` | Alpha | 1.17 | |
 | `CSIMigrationOpenStack` | `false` | Alpha | 1.14 | |
 | `CSIMigrationOpenStackComplete` | `false` | Alpha | 1.17 | |
+| `ConfigurableFSGroupPolicy` | `false` | Alpha | 1.18 | |
```

Comment:
There's also a section on this page that describes what the feature gate is

Reply:
fixed